web/src/components/storage/dasd/DASDTable.tsx

```diff
@@ -78,29 +80,58 @@ const columns = [
   { id: "formatted", label: _("Formatted") },
   { id: "partitionInfo", label: _("Partition Info") },
 ];
+const DevicesList = ({ devices }) => (
+  <ul>
+    {devices.map((d: DASDDevice) => (
+      <li key={d.id}>{d.id}</li>
+    ))}
+  </ul>
+);
+const FormatNotPossible = ({ devices, onAccept }) => (
+  <Popup isOpen title={_("Offline DASD devices")}>
+    <Text>
+      {_(
+        "The DASD devices listed below are offline and cannot be formatted, please unselect or activate them in order to continue",
+      )}
+    </Text>
+    <DevicesList devices={devices} />
+    <Popup.Actions>
+      <Popup.Confirm onClick={onAccept}>{_("Accept")}</Popup.Confirm>
+    </Popup.Actions>
+  </Popup>
+);
+
+const FormatConfirmation = ({ devices, onCancel, onConfirm }) => (
+  <Popup isOpen title={_("Format confirmation")}>
+    <Text>
+      {_("The DASD devices listed below are going to be formatted, do you want to proceed?")}
+    </Text>
```

Comment:
NP: Do you mind using a less generic title and more detailed body?

I'd go for something like

* Title: `Format selected devices?` or `Format following devices?`
* Body: 
 
   > This action will be triggered immediately and **will irreversibly destroy data** of devices listed below. Please, confirm you really want continue`

With a better wording for the body, but hope you got the idea:

*  A really informative title not describing the dialog itself
*  A useful body explaining the consequences if proceeding with the action. The idea is provide the user information to take the decision, not just putting a dialog in the middle to prevent them going ahead in the first click.

I know we are thinking in advanced users here, but I prefer to think globally when building the interface.